x-pack/plugins/cloud_security_posture/public/pages/configurations/layout/findings_distribution_bar.tsx

```diff
@@ -144,25 +132,16 @@ const DistributionBarPart = ({
   color: string;
   distributionOnClick: () => void;
   ['data-test-subj']: string;
+  ['aria-label']: string;
 }) => (
   <button
     data-test-subj={rest['data-test-subj']}
+    aria-label={rest['aria-label']}
     onClick={distributionOnClick}
-    css={css`
-      flex: ${value};
-      background: ${color};
-      height: 100%;
-    `}
+    style={{
```

Comment:
❓ I am curious why you decided to switch from `emotion-css` to `style`

Reply:
Since the bar has dynamic values for `flex` that might change on every interaction with the findings page according to the amount of passing and failed findings, I think we are not getting real benefit out of using Emotion here. For simplicity and to keep the code straightforward, we can just use the inline `style` prop. What do you think?

![image](https://github.com/elastic/kibana/assets/19270322/780ebd47-eed0-41e6-af06-d3f3db20e087)

Reply:
Actually, after some thought on it, we can still have the benefits of emotion if we move only the dynamic property to styles as for like this:

```tsx
 <button
    data-test-subj={rest['data-test-subj']}
    aria-label={rest['aria-label']}
    onClick={distributionOnClick}
    css={{
      background: color,
      height: '100%',
    }}
    style={{
      flex: value,
    }}
  />
```


I think that's actually preferred as we end up with classes that won't change on every interaction:

![image](https://github.com/elastic/kibana/assets/19270322/060037ab-8b20-4402-a8e4-6151d744b678)

I updated the code to reflect that let me know what you think